src/components/Slider/Slider.css

```diff
@@ -0,0 +1,4 @@
+.column-carousel {
```

Comment:
You will need a namespace there so that this classname doesn't affect other parts of the website:
```css
.Slider .column-carousel {
}
```
This classname needs to be added to `Slider.js` as well.